logstash-core/src/test/java/org/logstash/ackedqueue/TestSettings.java

```diff
@@ -9,7 +9,7 @@
 
 public class TestSettings {
 
-    public static Settings getSettings(int capacity) {
+    public static Settings volatileQueueSettings(int capacity) {
```

Comment:
lol - the real story is, I was reading some tests around `Queue.open()` and stumbled on `getSettingsCheckpointFilePageMemory()` and thought WTF is this setting?!?!  I seriously had NO idea what this settings was about :P